src/app/examples/list-example/list-example.component.html

```diff
@@ -1,3 +1,16 @@
+<!-- 
```

Comment:
I like that we're starting to explain functionality in the cookbook... looks good!